TestFoundation/TestNSURLSession.swift

```diff
@@ -164,7 +259,13 @@ class TestURLSession : XCTestCase {
     }
 }
 
-class DataTask: NSObject {
+#if os(Linux)
+typealias FoundationObject = NSObject
+#else
+typealias FoundationObject = SwiftFoundation.NSObject
+#endif
```

Comment:
This looks awkward. But I found the use of NSObject causing a "NSObject ambiguous for type lookup in this context" compiler error with Xcode.